velox/exec/OutputBuffer.h

```diff
@@ -192,6 +260,9 @@ class OutputBuffer {
   // the number of consumers, for example, increase number of TableWriter tasks.
   bool isOverutilized() const;
 
+  // Gets the OutputBufferStats of this output buffer.
+  OutputBufferStats getStats();
```

Comment:
stats() const

Reply:
Refactored to `stats()`, but the `stats()` function here could not be const because the mutex is used in lock guard.